gamification-evm-services/src/main/java/io/meeds/evm/gamification/service/BlockchainService.java

```diff
@@ -51,34 +54,40 @@
 import java.util.*;
 import java.util.stream.Stream;
 
-@Component
+@Service
 public class BlockchainService {
```

Comment:
Use a more specialized Bean name to avoid collision when it comes to define this as a `shared Servie` using annotation `@Service`

```suggestion
public class EvmBlockchainService {
```